CONTRIBUTING.md

```diff
@@ -60,60 +58,60 @@ for getting things done and landing your contribution.
 1. Click the fork button in the top right to clone the [nodejs.org repository](https://github.com/nodejs/nodejs.org/fork)
 2. Clone your fork using SSH, GitHub CLI, or HTTPS.
 
-    ```bash
-    git clone git@github.com:<YOUR_GITHUB_USERNAME>/nodejs.org.git # SSH
-    git clone https://github.com/<GITHUB_ID>/nodejs.org.git # HTTPS
-    gh repo clone <GITHUB_ID>/nodejs.org # GitHub CLI
-    ```
+   ```bash
+   git clone git@github.com:<YOUR_GITHUB_USERNAME>/nodejs.org.git # SSH
+   git clone https://github.com/<GITHUB_ID>/nodejs.org.git # HTTPS
+   gh repo clone <GITHUB_ID>/nodejs.org # GitHub CLI
```

Comment:
making this consistent with a suggestion from https://github.com/nodejs/nodejs.org/pull/5441


```suggestion
   gh repo clone <YOUR_GITHUB_USERNAME>/nodejs.org # GitHub CLI
```